AdtSampleApp/SampleClientApp/Log.cs

```diff
@@ -10,7 +8,7 @@ static public void Out(string s, ConsoleColor col = ConsoleColor.White)
         {
             Console.ForegroundColor = col;
             Console.WriteLine(s);
-            Console.ForegroundColor = ConsoleColor.White;
```

Comment:
Don't assume you know the console's original color. Either save it off to restore (before line 11), or just use ResetColor method.